### DISCUSSION TITLE
Generate a png in addition to the pdf and markdown files

### DISCUSSION THREAD

**Discussion by c2tz** [rendercv/rendercv#55] [Ideas]
Let me explain, I'd like rendercv to be able to generate a png in addition to the pdf and md files. I found this [example](https://github.com/GiovanniMenzano/cv) and it seems like a good idea because you could have a preview without having to download the pdf from a readme file on github. Does this sound feasible ?

**Comment by sinaatalay** [comment]
Yes, it’s easily doable. I already do something like this for automatic image generation for docs. I will implement it in my next availability. Thank you!

**Comment by c2tz** [comment]
https://github.com/sinaatalay/rendercv/issues/57
